typings/index.d.ts

```diff
@@ -97,6 +97,28 @@ declare namespace CodeceptJS {
   interface Globals {
     codeceptjs: typeof codeceptjs;
   }
+
+  class Helper {
+    constructor(config: unknown)
+    static _config(): unknown
+    _validateConfig(config: unknown): unknown
+    _setConfig(opts: unknown): unknown
+    _init(): unknown
+    _before(): unknown
+    _after(): unknown
+    _test(test: Mocha.Test): unknown
+    _passed(test: Mocha.Test): unknown
+    _failed(test: Mocha.Test): unknown
+    _beforeStep(step: CodeceptJS.Step): unknown
+    _afterStep(step: CodeceptJS.Step): unknown
+    _beforeSuite(suite: Mocha.Suite): unknown
+    _afterSuite(suite: Mocha.Suite): unknown
+    _finishTest(suite: Mocha.Suite): unknown
+    _useTo(description?: string, fn?: Function): void
```

Comment:
This should probably be typed as either `ReturnType<fn>` or `unknown` since the initial implementation in `@codeceptjs/helper` package returns the returned value of this function call.

Reply:
Agreed